### DISCUSSION TITLE
Closest Emergency Personel

### DISCUSSION THREAD

**Discussion by VaneRaklan** [jakobzhao/geog495#44] [Final Project]
**Application:** https://vaneraklan.github.io/ClosestEmergencyPersonel/index.html
**Repository:** https://github.com/VaneRaklan/ClosestEmergencyPersonel
**Description:** Users can search using the geocoder in the top right or click anywhere on the map and the application will mark the target location and highlight the nearest police station, fire station, and hospital as well as highlight the routes to take to each of them while displaying the travel time in minutes. The user can change the mode of transportation before clicking/searching in order to get their corresponding travel times and routes.
![SearchExample](https://user-images.githubusercontent.com/6405135/145126040-ce2e2a17-0f30-4702-9291-21e440d18b56.png)

**Comment by jakobzhao** [comment]
Well done! It is a very useful map for seattlite to plan for their travel to different facilities resources when suffering from any emergence.
two quick suggestions related to the design: 

1) it seems better to also show the icon of each facility to the console panel, then users can better understand the meaning of the icon on the map. 

2) the font size of the travel mode is smaller than that of the facility type.  You might want to keep them consistent.

**Comment by jacobf19** [comment]
This is a highly useful and really interesting map. I could see this being incredibly valuable for almost anyone. I was able to plug in my own address and was surprised to see the distances. One suggestion I have is that you include some information about the individual emergency sites. I also would suggest when listing the travel time, the decimals after the minute would be more useful if they were set to being seconds, rather than just hundredths of a minute because we count in 60 seconds not 100 seconds.

**Comment by zhuxiaoyu1019** [comment]
I really like your base map from both an aesthetic and practical perspective. The neon green highlights on the roads are an excellent way to illustrate the route in your application. Users are easily able to identify the dynamic highlight icons. The feature that allows a user to search for the nearest fire station, police station, and hospital based on a location on the map is powerful. I really believe it could be a really useful tool in the real world. 
The only suggestion I have is placing a respect icon beside each title in the sidebar would make it easier for new users to understand the application.

**Comment by notzulkifli** [comment]
The idea behind this is great, I find it very helpful that you are emphasizing the distances for the three locations (fire station, police station and hospital). Currently as of 12/09/21, the application has an issue when we click on the travel mode, it does not change any of the information that is being displayed or will be returned. Maybe adding minutes or hours for distance can be a feature you can add, or maybe changing the direction with mode of travel to something else so that the feature is efficient. Overall amazing job

**Comment by zeyany2** [comment]
Good Job! I really like this map. The idea is great and you did a great job cauculating the distance and showing the route. I am having trouble using travel model, is that something you are working on and will get it fixed later? And one suggestion, like other people said, having icons in side bar is a great idea. Otherwise, it's a great application to me!

**Comment by yodapancake** [comment]
Hi there. I really like the base map with the dark themed ground and green streets and also just the map in general. It's the most useful map I've seen so far in terms of providing me with actually important information. I also love how you can click anywhere and the map will route the user to each closest station. One thing I noticed is that when I tried to switch between methods of transportation in the menu after clicking somewhere and getting my routes, that the times to travel somewhere with a mode of transportation would not update from the initial calculation made with the transportation mode given at the start before clicking.
